### PR TITLE
test: add test to check forced update of scenario metadata parameters

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: cypress/integration/brewery/ScenarioMetadataParameters.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Login, ScenarioParameters } from '../../commons/actions';
```
Comment (Guismus) [inline cypress/integration/brewery/ScenarioMetadataParameters.spec.js:1]: Missing header at the start of file

Comment (csm-thu) [reply cypress/integration/brewery/ScenarioMetadataParameters.spec.js:4]: Done :+1:

**Review by nborde-CSM — APPROVED** [review]
One remark but LGTM :+1:

File: cypress/integration/brewery/ScenarioMetadataParameters.spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import {
+  SCENARIOS,
+  SOLUTIONS,
+  SCENARIO_METADATA_PARAMETERS_IDS,
+} from '../../fixtures/stubbing/ScenarioMetadataParameters';
+import { stub } from '../../commons/services/stubbing';
+import { apiUtils } from '../../commons/utils';
+
+const validateScenarioUpdateRequest = (req) => {
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/integration/brewery/ScenarioMetadataParameters.spec.js:13]: The mechanism to apply a validation function to the interception is already generic and implemented in the apiUtils file (the 2nd parameter of `interceptUpdateScenario`).
The function `validateScenarioUpdateRequest`, on the other hand, aims to check that the intercepted request contains the scenario metadata parameters: it is really specific to the current test and will probably never be necessary for another test.

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Pull request merged**